spec/models/event_schedule_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe EventSchedule, type: :model do
+  describe '.populate_upcoming_show_and_tell!' do
+    subject { described_class.populate_upcoming_show_and_tell! }
+    before { travel_to Date.new(2017, 1, 1) }
+    after { travel_back }
+
+    let(:fridays) do
```

Comment:
Usually if you're pushing things into an array you can simplify it to a map, something like:
`(0..3).map { |week| Date.new(2017, 1, 6 + (week * 7)) }`